blog/en/blog/2023/04/07/apisix-unity-group-q&a.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Unity Group's APISIX Experience: User-friendly, Robust, and Delightful"
+description: "Lukasz Biegaj, System Architect of Unity Group, shares the changes that APISIX has brought to their company in the interview."
+slug: "/blog/apisix-unity-group-q&a"
+category: "Case Studies"
+cover_url: ""
+published_at: "2023-04-11T00:00:00Z"
+tags: ["APISIX", "Unity Group", "AWS", "Separation of Roles"]
+author_name: Qi Zeng
+author_avatar_url: "https://avatars.githubusercontent.com/u/129834194?v=4"
+author_bio: ""
+canonical_url: ""
+---
+
```

Comment:
You'll have to redo this part and add `truncate` before merging.

![image](https://user-images.githubusercontent.com/36651058/231078232-930be8a6-21c3-41f7-997f-fccc21b41f23.png)

```
---
title: "Unity Group's APISIX Experience: User-friendly, Robust, and Delightful"
authors:
  - name: "Qi Zeng"
    title: "Author"
    url: "https://github.com/Qizeng-api7"
    image_url: "https://github.com/Qizeng-api7.png"
keywords: 
- APISIX
- Unity Group
- AWS
- Separation of Roles
description: Lukasz Biegaj, System Architect of Unity Group, shares the changes that APISIX has brought to their company in the interview.
tags: [Ecosystem]
---

> Lukasz Biegaj, System Architect of Unity Group, shares the changes that APISIX has brought to their company in the interview.

<!--truncate-->
```
See blog/zh/blog/2023/03/30/what-is-wasm-and-how-does-apache-apisix-support-it.md for details.